addons/account/models/account_move.py

```diff
@@ -3961,10 +3961,16 @@ def _post(self, soft=True):
         # reconcile if state is in draft and move has reversal_entry_id set
         draft_reverse_moves = to_post.filtered(lambda move: move.reversed_entry_id and move.reversed_entry_id.state == 'posted')
 
-        to_post.write({
-            'state': 'posted',
-            'posted_before': True,
-        })
+        to_post\
+            .with_context(
+                skip_invoice_sync=True,
+                skip_invoice_line_sync=True,
+                skip_account_move_synchronization=True,
+            )\
+            .write({
+                'state': 'posted',
+                'posted_before': True,
+            })
```

Comment:
```suggestion
        to_post.with_context(
            skip_invoice_sync=True,
            skip_invoice_line_sync=True,
            skip_account_move_synchronization=True,
        ).write({
            'state': 'posted',
            'posted_before': True,
        })
```

https://peps.python.org/pep-0008/#maximum-line-length
> The preferred way of wrapping long lines is by using Python’s implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. 
***These should be used in preference to using a backslash for line continuation.***